util/execdetails/execdetails.go

```diff
@@ -505,18 +545,22 @@ type CopRuntimeStats struct {
 	// have many region leaders, several coprocessor tasks can be sent to the
 	// same tikv-server instance. We have to use a list to maintain all tasks
 	// executed on each instance.
-	stats      map[string][]*BasicRuntimeStats
+	stats      map[string][]*basicCopRuntimeStats
 	scanDetail *ScanDetail
+	// do not use kv.StoreType because it will meet cycle import error
+	storeType string
 }
 
 // RecordOneCopTask records a specific cop tasks's execution detail.
 func (crs *CopRuntimeStats) RecordOneCopTask(address string, summary *tipb.ExecutorExecutionSummary) {
 	crs.Lock()
 	defer crs.Unlock()
 	crs.stats[address] = append(crs.stats[address],
-		&BasicRuntimeStats{loop: int32(*summary.NumIterations),
+		&basicCopRuntimeStats{BasicRuntimeStats: BasicRuntimeStats{loop: int32(*summary.NumIterations),
 			consume: int64(*summary.TimeProcessedNs),
-			rows:    int64(*summary.NumProducedRows)})
+			rows:    int64(*summary.NumProducedRows)},
+			threads:   int32(summary.GetConcurrency()),
```

Comment:
getConcurrency() -> getThreads()?

Reply:
`getConcurrency` is a method generated by protobuf